doc/api/globals.md

```diff
@@ -380,6 +380,15 @@ added: v0.0.1
 
 [`setTimeout`][] is described in the [timers][] section.
 
+## `DOMException`
+<!-- YAML
+added: REPLACEME
+-->
+
+<!-- type=global -->
+
+The WHATWG `DOMException` class. See [`DOMException`][] for more details.
```

Comment:
Maybe `Web IDL` instead of `WHATWG` ?

Reply:
https://dom.spec.whatwg.org/

hmmm, isn't it WHATWG DOMException?

Reply:
DOMException is defined here: https://heycam.github.io/webidl/#idl-DOMException (the DOM spec links to that too)
WHATWG is the group that writes the standards and it feels weird to me to use that name as the reference. Kind of like if we said "The TC39 SyntaxError".

Reply:
What confused me is that likely [`URL` has an IDL way](https://url.spec.whatwg.org/#idl-index):

```
[Exposed=(Window,Worker),
 LegacyWindowAlias=webkitURL]
interface URL {
  constructor(USVString url, optional USVString base);

  stringifier attribute USVString href;
  readonly attribute USVString origin;
           attribute USVString protocol;
           attribute USVString username;
           attribute USVString password;
           attribute USVString host;
           attribute USVString hostname;
           attribute USVString port;
           attribute USVString pathname;
           attribute USVString search;
  [SameObject] readonly attribute URLSearchParams searchParams;
           attribute USVString hash;

  USVString toJSON();
};

[Exposed=(Window,Worker)]
interface URLSearchParams {
  constructor(optional (sequence<sequence<USVString>> or record<USVString, USVString> or USVString) init = "");

  undefined append(USVString name, USVString value);
  undefined delete(USVString name);
  USVString? get(USVString name);
  sequence<USVString> getAll(USVString name);
  boolean has(USVString name);
  undefined set(USVString name, USVString value);

  undefined sort();

  iterable<USVString, USVString>;
  stringifier;
};
```

So when should we use Web IDL and when should we use WHATWG?

Reply:
I think we should stop using WHATWG completely and always use Web instead. I know that right now the situation is mixed. In the documentation we have "WHATWG URL" and "WHATWG Encoding", but on the other hand we have "Web Streams" and "Web Crypto"...

Reply:
You mean use:

`Web URL` / `Web TextEncoder`?

Reply:
I would just remove the `The WHATWG DOMException class.` sentence entirely.